### ISSUE TITLE
FFb12, v1.1.3: Don't display in context menu if text not editible.

### ISSUE THREAD

**Issue by MarkRH** [Dalimil/Find-and-Replace-Browser-Extension#2]
I noticed that if I highlight text in a page and right-click that this is displayed in the context menu.  Is there a way to disable this unless it's in a textarea or a toggle to turn it off as it's just wasting space in the context menu.  I suppose the other option is to just disable the extension until I actually want to use it.  

Anyway, a thought.

**Comment by Dalimil** [comment]
Yeah, unfortunately the Chrome API only allows me to create a new item in the context menu when the user selects some text (there's no way to differentiate between editable and non-editable using the API).
So one solution would be to remove the context menu entry completely (everywhere). I'll consider this...
